packages/block-library/src/post-tags/edit.js

```diff
@@ -34,9 +40,31 @@ function PostTagsDisplay() {
 	);
 }
 
-export default function PostTagsEdit() {
-	if ( ! useEntityId( 'postType', 'post' ) ) {
-		return __( 'Post Tags' );
+export default function PostTagsEdit( { context } ) {
+	if ( ! context.postType || ! context.postId ) {
+		return (
+			<Warning>
+				{ __( 'Post tags block: No post found for this block.' ) }
+			</Warning>
+		);
+
+		/**
+		 * Do not render the block when viewing a page (as opposed to a post)
+		 *
+		 * @todo By default, only posts can be grouped by tags. Therefore, without any configuration,
+		 * the post tags block will have no tags for pages. Plugins, however, can modify this behavior.
+		 * In the future, instead of only evaluating posts, we should check whether the
+		 * post_tag taxonomy is registered for the page post type.
```

Comment:
```suggestion
		 * post_tag taxonomy is registered for the current post type.
```

(This block could be used in other Custom Post Types as well.)

Reply:
Good catch! Making the change now.